cypress/e2e/settings/users/patron-block/maximum.charged.out.returned.cy.js

```diff
@@ -0,0 +1,204 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import TestTypes from '../../../../support/dictionary/testTypes';
+import devTeams from '../../../../support/dictionary/devTeams';
+import permissions from '../../../../support/dictionary/permissions';
+import UserEdit from '../../../../support/fragments/users/userEdit';
+import TopMenu from '../../../../support/fragments/topMenu';
+import SettingsMenu from '../../../../support/fragments/settingsMenu';
+import generateItemBarcode from '../../../../support/utils/generateItemBarcode';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../../../support/fragments/settings/users/patronGroups';
+import Location from '../../../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../../../support/fragments/users/users';
+import CheckInActions from '../../../../support/fragments/check-in-actions/checkInActions';
+import Checkout from '../../../../support/fragments/checkout/checkout';
+import ServicePoints from '../../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import Conditions from '../../../../support/fragments/settings/users/conditions';
+import Limits from '../../../../support/fragments/settings/users/limits';
+import UsersSearchPane from '../../../../support/fragments/users/usersSearchPane';
+
+function generateUniqueItemBarcodeWithShift(index = 0) {
+  return (generateItemBarcode() - Math.round(getRandomPostfix()) + '').substring(index);
+}
+
+describe('Patron Block: Maximum number of items charged out', () => {
+  const patronGroup = {
+    name: 'groupToPatronBlock' + getRandomPostfix(),
+  };
+  const userData = {
+    personal: {
+      lastname: null,
+    },
+  };
+  const itemsData = {
+    itemsWithSeparateInstance: [
+      { instanceTitle: `Instance ${getRandomPostfix()}` },
+      { instanceTitle: `Instance ${getRandomPostfix()}` },
+      { instanceTitle: `Instance ${getRandomPostfix()}` },
+      { instanceTitle: `Instance ${getRandomPostfix()}` },
+      { instanceTitle: `Instance ${getRandomPostfix()}` },
+    ],
+  };
+  const testData = {
+    userServicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(
+      'autotest receive notice triggers',
```

Comment:
please rename the servicepoint's name to "autotest charged out limit"

Reply:
Done